clients/vscode-hlasmplugin/proc_grps_schema

```diff
@@ -48,21 +48,56 @@
                     "asm_options": {
                         "type": "object",
                         "description": "List of assembler options",
-                        "properties": {
-                            "SYSPARM": {
-                                "type": "string",
-                                "description": "It specifies the character string the assembler assigns to the &SYSPARM system variable symbol.",
-                                "maxlength": 255
-                            },
-                            "PROFILE": {
-                                "type": "string",
-                                "description": "Profile Member to be copied into the source program."
-                            },
-                            "SYSTEM_ID": {
-                                "type": "string",
-                                "description": "Provides the value for the SYSTEM_ID system variable. Defaults to 'z/OS 02.04.00' when omitted."
+                      "properties": {
+                        "OPTABLE": {
+                          "type": "string",
+                          "description": "Specifies the instruction set to use.",
+                          "anyOf": [
+                            {
+                              "type": "string",
+                              "description": "OPTABLE value",
+                              "enum": [
```

Comment:
Is there a benefit to this form compared to the plain enum at the description level?

Reply:
Probably not. I just didn't realize that it could be done without `anyOf`